src/main/java/de/tum/in/www1/artemis/service/connectors/gitlab/GitLabUserManagementService.java

```diff
@@ -137,7 +137,7 @@ public void updateCoursePermissions(Course updatedCourse, String oldInstructorGr
         }
 
         final List<ProgrammingExercise> programmingExercises = programmingExerciseRepository.findAllProgrammingExercisesInCourseOrInExamsOfCourse(updatedCourse);
-        log.info("Update Gitlab permissions for programming exercises: " + programmingExercises.stream().map(ProgrammingExercise::getProjectKey).toList());
+        log.info("Update Gitlab permissions for programming exercises: {}", programmingExercises.stream().map(ProgrammingExercise::getProjectKey).toList());
```

Comment:
in this entire class: Gitlab ---> GitLab

Reply:
Gitlab is used in many places, a change like this is out of scope of this PR.
![grafik](https://user-images.githubusercontent.com/26540346/178591792-f7bd3e27-ac10-4ffc-a95d-0e4c98300932.png)

Reply:
But this is out of scope for this PR, we could consider it in a follow-up.